Application/Frontend/pages/api/add_freind_request_to_user.ts

```diff
@@ -0,0 +1,42 @@
+import type { NextApiRequest, NextApiResponse } from 'next';
+import { MongoClient, ObjectId } from 'mongodb';
+import { getMongoDbUri } from '@/lib/dbConfig';
+
+export default async function handler(req: NextApiRequest, res: NextApiResponse) {
+  if (req.method === 'POST') {
+    const { id, requestingUserId } = req.body; // Now also receiving the ID of the user sending the friend request
+
+    let client: MongoClient | null = null;
+
+    try {
+      client = new MongoClient(getMongoDbUri());
+      await client.connect();
+      const db = client.db('Accord');
+      const accountsCollection = db.collection("Test_friend_list_request_insertion");
```

Comment:
You also don't need to create a test collection for this particular case as you can use the Accounts collection directly. You can just create two accounts with the provider to test things that require two users. If you are testing fields that don't currently exist in any document in the collection, you can just add them to existing collections as long as they don't break anything or create the new fields as part of account syncing (in the registration endpoint)